setup.py

```diff
@@ -19,7 +19,8 @@
     "multiprocessing-logging",
     "psutil",
     "configobj",
-    "tensorflow>=2.5.0",
+    'tensorflow>=2.5.0;  platform_system!="Darwin" or platform_machine!="arm64"',
+    'tensorflow-macos>=2.5.0;  platform_system=="Darwin" and platform_machine=="arm64"',
```

Comment:
This appears to work and passes `pre-commit`

```suggestion
    "tensorflow>=2.5.0;  "
    + 'platform_system!="Darwin" or platform_machine!="arm64"',
    "tensorflow-macos>=2.5.0;  "
    + 'platform_system=="Darwin" and platform_machine=="arm64"',
```

Reply:
I really *hate* how that looks, reduces readability and understanding IMO
@adamltyson ?
What about:
```
"tensorflow>=2.5.0;  platform_system!='Darwin' "
                     + " or platform_machine!='arm64'",
```
Or something along those lines? Also horrific, but less so maybe?

Reply:
I doubt `black` would approve of that though ^

Reply:
is this any better?

```suggestion
    "tensorflow>=2.5.0; "
    + 'platform_system!="Darwin" or '
    + 'platform_machine!="arm64"',
    "tensorflow-macos>=2.5.0; "
    + 'platform_system=="Darwin" and '
    + 'platform_machine=="arm64"',
```

Reply:
Or better still - swapping `'` & `"`

```suggestion
    "tensorflow>=2.5.0; "
    + "platform_system!='Darwin' or "
    + "platform_machine!='arm64'",
    "tensorflow-macos>=2.5.0; "
    + "platform_system=='Darwin' and "
    + "platform_machine=='arm64'",
```

Reply:
swapping the " and ' makes sense. making even more lines seems even less understandable?

Reply:
I mean I don't case 🤣 
Up to @adamltyson

Reply:
I actually prefer the multiple lines 🤷  makes it more clear what the `or`/`and` are referring to